src/Layer/C3DTilesLayer.js

```diff
@@ -194,7 +193,8 @@ class C3DTilesLayer extends GeometryLayer {
         });
     }
 
-    preUpdate() {
+    preUpdate(context) {
+        this.scale = context.camera._preSSE;
```

Comment:
This code shall be in `pre3dTilesUpdate`.

Reply:
I did it, I had to carry around the context value though